src/Aspire.Hosting/Dashboard/DashboardLifecycleHook.cs

```diff
@@ -36,6 +42,8 @@ public Task BeforeStartAsync(DistributedApplicationModel model, CancellationToke
             AddDashboardResource(model);
         }
 
+        _ = SubscribeDashboardLogsAsync();
```

Comment:
Can we store the task and await it when shutting down? Log unexcepted errors from the task.

Reply:
Done